### ISSUE TITLE
p-value calculation is incorrect for one-tailed tests + need to implement two-tailed

### ISSUE THREAD

**Issue by hendersontrent** [hendersontrent/correctipy#21]

**Comment by zexuanhao** [comment]
It is inappropriate to decide which tail's p-value to use based on the sign of the observed statistic. It is recommended to add a "_tails_" parameter to the function, allowing the user to choose between one-tailed or two-tailed tests.

**Comment by hendersontrent** [comment]
Yes I know. The correct operation is implemented in my correctR package https://github.com/hendersontrent/correctR but I honestly forget that this Python version exists. Will update sometime
